packages/stylehacks/src/plugin.js

```diff
@@ -1,59 +1,53 @@
-export default function plugin(targets, nodeTypes, detect) {
```

Comment:
I think this looks like a custom implementation of subclassing. The function returns creates a closure to add custom versions of `targets`, `nodeTypes` and `detect` to the `Plugin` class.